wagtail/admin/templates/wagtailadmin/panels/comments/comment_panel.html

```diff
@@ -1,5 +1,5 @@
 {% load wagtailadmin_tags %}
-<button type="button" id="comment-icon" data-annotation class="w-field__comment-button w-field__comment-button--reveal u-hidden">
+<button type="button" id="comment-icon" class="w-field__comment-button w-field__comment-button--reveal u-hidden" {% include "wagtailadmin/shared/attrs.html" with attrs=self.attrs %}>
```

Comment:
best to pass `attrs` in the context data. using `self` can introduce incompatibilities. see #10538 as an example

Reply:
Yes that wasn't seems to working i dont know for some reason 

@lb- found this way which we are initially using just for the sake of the progress we would definitely need to find out some what was happening

Reply:
you need to add it via the panel's `BoundPanel.get_context_data()`